lib/core/utils/date_util.dart

```diff
@@ -1,4 +1,9 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:live_score/core/constants/dashboard_constants.dart';
+
 class DateUtil {
```

Comment:
This util class needs to be refactored. Please move the static functions to an extension on DateTime. Put scroll controller and it's logic in appropriate widgets.

Reply:
And delete this file afterwards